xrpl_types/src/currency.rs

```diff
@@ -1,10 +1,10 @@
-use serde::Serialize;
+use serde::{Serialize, Serializer};
 
 /// An XRP Ledger currency. Can be either an Issued Currency (IOU) or the native
 /// XRP digital asset.
-#[derive(Clone, Serialize)]
+#[derive(Debug, Clone)]
 pub enum Currency {
-    Issued { name: String, issuer: String },
```

Comment:
Hm, I don't really like using 'currency' here, but, alas it's whats used in xrpl.org docs, so I am OK with this change also.

Reply:
Yes I though using the "official" term is better. Also because `name` indicates something displayable. And it does not need to be. It can be a 20 byte hex encoding like `434F524500000000000000000000000000000000` for the CORE token.

Reply:
On xrplscan it is called "Token code", but in docs it is referred to as currency. 
![image](https://github.com/gmosx/xrpl_sdk_rust/assets/110611023/d2f873ee-0275-4452-bcb0-16fb1171c486)

It could also be named `currency_code`. https://xrpl.org/currency-formats.html#currency-codes

Reply:
Hmm, we may consider currency_code (or just 'code') for a future PR.